plugins/pulp_docker/plugins/importers/importer.py

```diff
@@ -295,10 +303,8 @@ def _import_manifest(conduit, unit, dest_repo):
         """
         units_added = set()
 
-        # Add manifests and catalog referenced blobs
+        # Catalog referenced blobs
```

Comment:
Since the word "Catalog" has at least one other specific meaning now in Pulp 2, it would be best to avoid its use here.